backend/src/services/dish.ts

```diff
@@ -1,5 +1,82 @@
+import { DecodedIdToken } from 'firebase-admin/lib/auth/token-verifier'
 import { Dish, DishStatus, DishTableVM } from '../models/dish'
 import { Transaction } from '../models/transaction'
+import { db } from './firebase'
+
+export function getAllUserDishes(
```

Comment:
Can you also add logs in the helper functions for easier reading and better understanding? because there are no comments with code so its hard for someone to understand the usage of the function/code